src/contrib/cluster/Akka.Cluster.Sharding/ClusterSharding.cs

```diff
@@ -595,7 +784,7 @@ private void Done(bool ok)
     /// Check if we've received a shard start request.
     /// </summary>
     [Serializable]
-    public sealed class ResendShardHost
+    internal sealed class ResendShardHost
```

Comment:
I'm not sure I get it. AFAIK protocol buffers are able to serialize hidden types.